client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-identify/test/index.tsx

```diff
@@ -1,8 +1,6 @@
 /**
  * @jest-environment jsdom
  */
-// @ts-nocheck - TODO: Fix TypeScript issues
```

Comment:
We'll need to bring this back, or fix the TS errors in the meantime.

Reply:
I see TypeScript complaining about `.mockReturnValue`. I think that all you need to fix this is to wrap the offending functions with `jest.mocked`. It returns the same function, but with a correct type (`jest.MockedFunction<T>`) so that `.mockReturnValue` can be called on it.

Reply:
Fixed the TypeScript error in the rebased 0a753852397.